lib/config.js

```diff
@@ -130,6 +134,10 @@ const conf = convict({
     port: {
       default: '3306',
       env: 'MYSQL_PORT'
+    },
+    connectionLimit: {
```

Comment:
Ah okay, both this and `maxSockets` had the same doc `The maximum number of connections that the pool can use at once.` so it was a bit confusing